src/components/Map.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useRef, useEffect, useMemo } from 'react';
+import mapboxgl from 'mapbox-gl';
+
+const ACCESS_TOKEN = 'pk.eyJ1Ijoibmljb2xhc21hcnRpbmV6MDUxMCIsImEiOiJja3cwbW84Z2VkaHhxMnZwZzIyODg5Nzk4In0.3WRMNapO4tRArFLEoenJKQ'
+
+mapboxgl.accessToken = ACCESS_TOKEN;
+
+function Map({ className = '', latitude = -34.599722, longitude = -58.381944}) {
+    const mapContainer = useRef(null);
+    const map = useRef(null);
+
+    const nav = useMemo(() => new mapboxgl.NavigationControl(), []);
+
+    const geolocate = useMemo(() => new mapboxgl.GeolocateControl({
+        positionOptions: {
+            enableHighAccuracy: true
+        },
+        trackUserLocation: true
+    }), []);
+
+    useEffect(() => {
+        if (map.current) return;
+        map.current = new mapboxgl.Map({
+            container: mapContainer.current,
+            style: 'mapbox://styles/mapbox/streets-v11',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
            style: "mapbox://styles/mapbox/streets-v11",
```